diffhtwo/experimental/halpha_luminosity.py

```diff
@@ -52,20 +52,61 @@ def get_L_halpha(
 )
 
 
+def safe_log10(x):
```

Comment:
Nice. Little tricks like this are a big part of becoming an expert JAX practitioner. This little kernel should be jitted though.